modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/news/index.html

```diff
@@ -0,0 +1,28 @@
+<div class="border-neutral-3 d-flex evp_news_container p-3">
```

Comment:
Please, change the class name here.
```suggestion
<div class="border-neutral-3 d-flex evp-news-container p-3">
```